### PR TITLE
FEATURE: ONE-4413 Resize all columns of given measure

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**
Allows resize all columns of given measure at once when resized with Alt key pressed

---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/3207
- gdc-dashboards: https://github.com/gooddata/gdc-dashboards/pull/2923

# Related Jira tasks
- ONE-4413: https://jira.intgdc.com/browse/ONE-4413

### CODE REVIEW

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7883/ : FAILURE in 2m 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4063/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7887/ : FAILURE in 6m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4067/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7892/ : FAILURE in 7m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4072/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7896/ : SUCCESS in 6m 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4073/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7904/ : SUCCESS in 9m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4084/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7905/ : SUCCESS in 7m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4085/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7906/ : SUCCESS in 4m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4086/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7907/ : SUCCESS in 4m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4087/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7908/ : FAILURE in 1m 34s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4088/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7910/ : FAILURE in 1m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4090/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7912/ : FAILURE in 1m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4092/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7913/ : FAILURE in 1m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4093/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7914/ : FAILURE in 7m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4094/ : SUCCESS in 6s

**Comment by ivan-nejezchleb** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7915/ : SUCCESS in 8m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4095/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by ivan-nejezchleb** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7916/ : SUCCESS in 4m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4097/ : SUCCESS in 4s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4141/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4098/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3509/ : UNSTABLE in 12m 04s

**Comment by ivan-nejezchleb** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7917/ : SUCCESS in 2m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4099/ : SUCCESS in 3s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4142/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4100/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3510/ : SUCCESS in 12m 07s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7918/ : SUCCESS in 6m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4101/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7919/ : SUCCESS in 4m 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4102/ : SUCCESS in 3s

File: src/components/core/pivotTable/ResizedColumnsStore.ts [hunk 1]
```diff
@@ -1,24 +1,31 @@
 // (C) 2007-2020 GoodData Corporation
-// tslint:disable:prefer-conditional-expression
```
Comment (pbenes) [inline src/components/core/pivotTable/ResizedColumnsStore.ts:2]: :+1:

File: src/components/core/pivotTable/ResizedColumnsStore.ts [hunk 2]
```diff
@@ -50,6 +66,12 @@ export class ResizedColumnsStore {
             return this.convertItem(this.manuallyResizedColumns[colId]);
         }
 
+        const weakColumnWidth = this.getMatchedWeakMeasuresColumnWidths(item);
```
Comment (pbenes) [inline src/components/core/pivotTable/ResizedColumnsStore.ts:69]: It seems to me that singular in the name of method would be more expressive, i.e. `getMatchedWeakMeasuresColumnWidth(..)` as it returns only one record

File: src/components/core/pivotTable/ResizedColumnsStore.ts [hunk 3]
```diff
@@ -93,6 +142,28 @@ export class ResizedColumnsStore {
...
+            );
+            const colIds = Object.keys(this.manuallyResizedColumns);
+            colIds.forEach(colId => {
+                const item = this.manuallyResizedColumns[colId];
+                if (
+                    isColumnWidthAuto(item.width) &&
+                    this.isMatchingWeakWidth(item, weakColumnWidth) &&
+                    !this.isAllMeasureColumWidthUsed()
+                ) {
+                    this.manuallyResizedColumns = omit(this.manuallyResizedColumns, colId);
```
Comment (pbenes) [inline src/components/core/pivotTable/ResizedColumnsStore.ts:163]: It's just a minor thing but what about avoiding forEach and using https://lodash.com/docs/#omitBy ? Something like (not tested)
```
this.manuallyResizedColumns = omitBy(this.manuallyResizedColumns,  item => 
    isColumnWidthAuto(item.width) &&
    this.isMatchingWeakWidth(item, weakColumnWidth) &&
    !this.isAllMeasureColumWidthUsed()
);
```

File: src/components/core/pivotTable/ResizedColumnsStore.ts [hunk 4]
```diff
@@ -138,48 +213,104 @@ export class ResizedColumnsStore {
...
+            return columnWidths.filter(
+                item => isAttributeColumnWidthItem(item) || isMeasureColumnWidthItem(item),
+            );
+        }
+        return [];
+    }
+
+    private filterWeakColumnWidthItems(columnWidths: ColumnWidthItem[]): IWeakMeasureColumnWidthItemsMap {
+        if (columnWidths) {
+            const onlyWeakWidthItems: IWeakMeasureColumnWidthItem[] = columnWidths.reduce(
```
Comment (pbenes) [inline src/components/core/pivotTable/ResizedColumnsStore.ts:260]: Consider simplifying to `.filter(isWeakMeasureColumnWidthItem)`

File: src/components/core/pivotTable/ResizedColumnsStore.ts [hunk 5]
```diff
@@ -138,48 +213,104 @@ export class ResizedColumnsStore {
...
+                (result: IWeakMeasureColumnWidthItem[], item: ColumnWidthItem) => {
+                    if (isWeakMeasureColumnWidthItem(item)) {
+                        result.push(item);
+                    }
+                    return result;
+                },
+                [],
+            );
+            return onlyWeakWidthItems.reduce(
```
Comment (pbenes) [inline src/components/core/pivotTable/ResizedColumnsStore.ts:269]: consider simplifying to 
```
.reduce((map, weakWidthItem) => ({ 
  ...map, 
  [weakWidthItem.measureColumnWidthItem.locator.measureLocatorItem.measureIdentifier]: weakWidthItem 
})
```

File: src/components/core/pivotTable/tests/ResizedColumnsStore.spec.ts [hunk 6]
```diff
@@ -130,69 +149,264 @@ describe("ResizedColumnsStore", () => {
...
+                {
+                    a_4DOTdf: { width: { value: 200 } },
+                },
+                42,
+                weakMeasuresColumnWidths,
+            );
+
+            const columnMock = getFakeColumn({
+                colId: "m_0",
+                type: "MEASURE_COLUMN",
```
Comment (pbenes) [inline src/components/core/pivotTable/tests/ResizedColumnsStore.spec.ts:231]: It's sometimes used as `type: MEASURE_COLUMN` and sometimes as `type: "MEASURE_COLUMN"`. Is it an intention not to use imported constant?

Comment (ivan-nejezchleb) [reply src/components/core/pivotTable/tests/ResizedColumnsStore.spec.ts:231]: fixed

File: src/components/core/pivotTable/ResizedColumnsStore.ts (hunk 5 above)
Comment (ivan-nejezchleb) [reply src/components/core/pivotTable/ResizedColumnsStore.ts:269]: fixed

File: src/components/core/pivotTable/ResizedColumnsStore.ts (hunk 4 above)
Comment (ivan-nejezchleb) [reply src/components/core/pivotTable/ResizedColumnsStore.ts:260]: done

File: src/components/core/pivotTable/ResizedColumnsStore.ts (hunk 3 above)
Comment (ivan-nejezchleb) [reply src/components/core/pivotTable/ResizedColumnsStore.ts:163]: done

File: src/components/core/pivotTable/ResizedColumnsStore.ts (hunk 2 above)
Comment (ivan-nejezchleb) [reply src/components/core/pivotTable/ResizedColumnsStore.ts:69]: done

File: examples/src/components/PivotTableSizingWithMultipleMeasuresExample.jsx [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+
+const isMeasureColumnWidthItem = columnWidthItem => {
+    return (
+        columnWidthItem &&
+        columnWidthItem.measureColumnWidthItem !== undefined &&
+        columnWidthItem.measureColumnWidthItem.locators !== undefined
+    );
+};
+
+const isLocatorsEqual = (locator1, locator2) => {
```
[the remark this answers is not in the document]
Comment (ivan-nejezchleb) [reply examples/src/components/PivotTableSizingWithMultipleMeasuresExample.jsx:54]: done

File: src/components/core/PivotTable.tsx [hunk 8]
```diff
@@ -1055,6 +1055,7 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
             this.resizedColumnsStore.addToManuallyResizedColumn(column, true);
             column.getColDef().suppressSizeToFit = false;
         }
+        column.getColDef().suppressSizeToFit = false;
```
[the remark this answers is not in the document]
Comment (ivan-nejezchleb) [reply src/components/core/PivotTable.tsx:1058]: fixed

**Comment by ivan-nejezchleb** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1690.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7921/ : SUCCESS in 5m 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4103/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4104/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3511/ : FAILURE in 27m 56s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7926/ : SUCCESS in 8m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4115/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7927/ : SUCCESS in 3m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4117/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7929/ : SUCCESS in 4m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4121/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7931/ : FAILURE in 2m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4123/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7932/ : FAILURE in 1m 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4124/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7936/ : SUCCESS in 4m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4129/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7937/ : SUCCESS in 3m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4130/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4144/ : UNSTABLE in 9m 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/145/ : SUCCESS in 17s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7938/ : SUCCESS in 3m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1923/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2887/ : SUCCESS in 18m 41s

**Comment by pbenes** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4145/ : UNSTABLE in 16m 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/146/ : SUCCESS in 20s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7943/ : SUCCESS in 5m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1924/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2888/ : SUCCESS in 17m 55s

**Comment by patrikbraborec** [comment]
extended test - storybook

**Comment by patrikbraborec** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7944/ : SUCCESS in 3m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4136/ : SUCCESS in 3s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4146/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4137/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3523/ : SUCCESS in 9m 45s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4147/ : UNSTABLE in 10m 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/147/ : SUCCESS in 19s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7945/ : SUCCESS in 4m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1925/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2889/ : SUCCESS in 19m 03s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7946/ : FAILURE in 1m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4138/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7947/ : SUCCESS in 2m 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/4139/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4149/ : SUCCESS in 11m 02s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/149/ : SUCCESS in 20s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7959/ : SUCCESS in 3m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1927/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2896/ : FAILURE in 19m 41s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4150/ : SUCCESS in 12m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/150/ : SUCCESS in 16s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7960/ : SUCCESS in 6m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1928/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2897/ : SUCCESS in 20m 04s

**Pull request merged**